blueocean-personalization/package.json

```diff
@@ -9,7 +9,8 @@
     "test": "gulp test",
     "test:watch": "gulp test:watch",
     "bundle": "gulp bundle",
-    "bundle:watch": "gulp bundle:watch"
+    "bundle:watch": "gulp bundle:watch",
+    "postinstall": "node ../bin/cleanshrink.js"
```

Comment:
This works fine in the docker environment which actually surprised me. You can actually see this for yourself at the link below, just search for "cleanshrink":

https://ci.blueocean.io/blue/rest/organizations/jenkins/pipelines/blueocean/branches/speed-up-builds/runs/38/log/?start=0

I suspect that `maven-frontend-plugin` is setting up a path that has `node` on it when it executes `npm install`? In which case it should work fine, it might only fail if someone calls `npm install` from their own command line... but if the user has `npm` installed then surely that have `node` available globally too?

Not sure what the proper would fix be unless we changed it to:

`"postinstall": "./node/node ../bin/cleanshrink.js"`

but of course that would fail if a user hadn't run the Maven build at least once to get a local node installed. I guess I'm inclined just to assume that node is available; I'm not really worried about this script running under different versions of node (such as a different version that the user might have installed locally) because the script is pretty vanilla JS.